randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -985,9 +992,19 @@ Extra - minimap_data: [{'x': 70, 'y': 44}, {'x': 70, 'y': 45}, {'x': 70, 'y': 46
           Space Jump or Walljump (Expert) or Can Use Spider Ball
           # IBJ
           Infinite Bomb Jumping (Advanced) and Can Use Bombs
-          All of the following:
-              Walljump (Beginner)
-              Hi-Jump Boots or Enabled Septogg Helpers
+          Walljump (Beginner) and Enabled Septogg Helpers
```

Comment:
This is missing HJ now. The septogg doesn't spawn if you have it.

Reply:
Change made. But should the Septogg not stay regardless of items? I thought that was a change that was planned?

Reply:
you're right, my mind is still too much in vanilla, whoops.
Disregard this whole comment then.

Reply:
I think the original requirements are fine since you can use either HJ or Septoggs with Walljump. More of a side note that the Septogg shouldn't disappear at all.

Reply:
yeah, you're right again. If HJ is excluded, it means that with disabled septoggs, you wont have any logical path up with HJ, even tho its clearly possible